src/components/Dropdown/_Dropdown.scss

```diff
@@ -0,0 +1,80 @@
+.dropdown {
+  color: inherit;
+  display: inline-block;
+
+  &__content {
+    background-color: var(--ui-white);
+    border: 1px solid var(--ui-gray-medium);
+    display: none;
+    padding: var(--space-xxs) var(--space-s) var(--space-xs);
+    position: absolute;
+    transition: all 0.35s;
+    width: 259px;
+    z-index: 1;
+  }
+
+  &__label {
+    @include button--outline;
+
+    align-items: center;
+    border: 1px solid var(--ui-gray-medium);
+    cursor: pointer;
+    font-weight: 300;
+    justify-content: space-between;
+    width: 148px;
+
+    .icon--minus {
+      display: none;
+    }
+
+    &:hover {
+      background-color: var(--ui-gray-light);
+    }
+  }
+
+  &__input {
+    opacity: 0;
+    position: absolute;
+    z-index: -1;
+
+    &:checked {
+      + .dropdown__label .icon--minus {
+        display: inline;
+      }
+
+      + .dropdown__label .icon--plus {
+        display: none;
+      }
+
+      ~ .dropdown__content {
+        background-color: #ffffff;
+        display: inline-block;
+        // compensate for absence of border-bottom on label
+        margin-top: -1px;
+        position: absolute;
+      }
+    }
+  }
+
+  // Style label in open state
+  &__input:checked + .dropdown__label {
+    background-color: var(--ui-gray-light);
+    border-bottom: 0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+    // compensate for absence of border-bottom
+    margin-bottom: 1px;
+  }
+
+  &--fieldset {
+    fieldset {
```

Comment:
Let's give each of these selectors BEM classnames instead of styling the elements themselves.